plutus-playground-server/usecases/Starter.hs

```diff
@@ -0,0 +1,84 @@
+{-# LANGUAGE DataKinds           #-}
+{-# LANGUAGE DeriveAnyClass      #-}
+{-# LANGUAGE DeriveGeneric       #-}
+{-# LANGUAGE FlexibleContexts    #-}
+{-# LANGUAGE NoImplicitPrelude   #-}
+{-# LANGUAGE OverloadedStrings   #-}
+{-# LANGUAGE RecordWildCards     #-}
```

Comment:
This isn't needed as far cas I can tell (also I've been meaning to remove `RecordWildCards` from the sample contracts altogether)

Reply:
I will need the NoImplicitPrelude for error.